error during build: BuildOperationType.java unmappable character (0x9D) for encoding windows-1252

<!--- 
Please follow the instructions below. 
We receive dozens of issues every week, so to stay productive, we will close issues that don't provide enough information. 

Please open Android-related issues on the Android Issue Tracker at https://source.android.com/source/report-bugs
Please open Gradle Native-related issues at https://github.com/gradle/gradle-native/issues
-->

<!--- Provide a brief summary of the issue in the title above -->

There's something about the quote ` “process”` that causes an error.

https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/build-operations/src/main/java/org/gradle/internal/operations/BuildOperationType.java#L58

 
```
> Task :build-operations:javadoc FAILED 
\gradle\gradle\subprojects\build-operations\src\main\java\org\gradle\internal\operations\BuildOperationType.java:58: error: 
unmappable character (0x9D) for encoding windows-1252
 * completed, in order to ÔÇ£process´┐¢? them on a separate thread.                        
```
 

If I replace it with 'normal' quotes the error seems to go away.

### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

Gradle repo at this commit: https://github.com/gradle/gradle/commit/36cad0c3425102540db84b80ec85702f69b948a0

Windows 10.

<!--- A build scan https://scans.gradle.com/get-started is ideal -->
Build scan URL: I can't run a build scan because the Gradle repo is too resource intensive and the tests are too flaky

--- a/subprojects/build-operations/src/main/java/org/gradle/internal/operations/BuildOperationType.java
+++ b/subprojects/build-operations/src/main/java/org/gradle/internal/operations/BuildOperationType.java
@@ -55,7 +55,7 @@ package org.gradle.internal.operations;
  *
  * The lifecycle of details and result objects are effectively undetermined.
  * The build scan plugin will retain the objects for a short time after the operation has
- * completed, in order to “process” them on a separate thread.
+ * completed, in order to "process" them on a separate thread.
  * It can be assumed that this processing happens relatively quickly,
  * after which the objects are no longer retained.
  *

